src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/PrivateDataRetrievalService.java

```diff
@@ -135,6 +138,7 @@ public PrivateBoDataListApi getBeneficialOwnersData(String companyNumber)
                         logMap);
                 return new PrivateBoDataListApi(Collections.emptyList());
             }
+            ApiLogger.errorContext(String.format("Error retrieving Beneficial Owners for Company Number %s", companyNumber), e);
```

Comment:
This seems to be a duplicate of the log on line 144

Reply:
The message is similar although I purposely worded it slightly differently. The message on line 144 is in a different `catch` block.